.github/workflows/build.yml

```
@@ -11,35 +11,30 @@ jobs:
     name: Analyze with SonarCloud
     runs-on: ubuntu-latest
     steps:
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)